android/lib/src/main/java/com/ern/api/impl/core/ElectrodeReactFragmentActivityDelegate.java

```diff
@@ -33,6 +29,18 @@
     protected FragmentActivity mFragmentActivity;
     private DataProvider dataProvider;
 
+
+    private boolean upEnabledForRoot;
```

Comment:
Since this is a member variable, can you rename it to `mUpEnabledForRoot`? Then you can also get rid of `this.` on line 41. When you make the change, could you also remove the extra newline above (line 32)?

Reply:
will do. thanks.